internal/server/middleware.go

```diff
@@ -80,7 +80,25 @@ func authenticatedMiddleware(srv *Server) gin.HandlerFunc {
 				return
 			}
 
-			c.Request = c.Request.WithContext(data.WithOrg(c.Request.Context(), authned.Organization))
+			org := data.OrgFromContext(c.Request.Context())
+
+			if org == nil {
+				org = authned.Organization
```

Comment:
This reads a little weird. Maybe:
```
if org == nil {
  if authned.Organization != nil {
    org = authned.Organization
  } else {
    ... (Org is req'd)
    return
  }
}
```

Reply:
is it possible that authned.Organization could ever be missing? what if the org is deleted after the key is issued?